modules/ossm-configuring-jaeger.adoc

```diff
@@ -31,101 +30,148 @@ The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as
-            nodeCount: 3
-            redundancyPolicy:
-            resources:
-              requests:
-                memory: "16Gi"
-                cpu: "1"
-              limits:
-                memory: "16Gi"
+      enabled: true
+      ingress:
+        enabled: true
+      jaeger:
+        template: production-elasticsearch
+        elasticsearch:
+          nodeCount: 3
+          redundancyPolicy:
+          resources:
+            requests:
+              cpu: "1"
+              memory: "16Gi"
+            limits:
+              cpu: "1"
+              memory: "16Gi"
+
 ----
 
 .Elasticsearch parameters
 [options="header"]
-[cols="l, a, a"]
-|===
-|Parameter |Values |Description
-|nodeCount
-|integer value
-|Number of Elasticsearch nodes
-
-|cpu
-|Specified in units of cores (e.g., 200m, 0.5, 1)
-|Number of central processing units
-
-|memory
-|Specified in units of bytes (e.g., 200Ki, 50Mi, 5Gi)
-|Memory limit
-|===
-
-.Sample configurations
-[options="header"]
-[cols="l, a, a"]
+[cols="l, a, a, a, a"]
 |===
-| Parameter |Proof of Concept | Minimal Deployment
+|Parameter |Description |Values |Default Value |Examples
 
-|Node count
-|1
-|3
+|tracing:
+  enabled
+|This parameter enables/disables tracing in {ProductShortName}. Jaeger is installed by default.
+|`true`/`false`
+|`true`
+|
 
-|Requests CPU
-|500m
+|ingress:
+  enabled
+|This parameter enables/disables ingress for Jaeger.
+|`true`/`false`
+|`true`
+|
+
+|jaeger
+   template
+|This parameter specifies which Jaeger deployment strategy to use.
+|`all-in-one`/`production`
+|`all-in-one`
+|
+
+|elasticsearch:
+  nodeCount
+|Number of Elasticsearch nodes to create.
+|Integer value.
 |1
+|Proof of concept = 1,
+Minimum deployment =3
 
-|Requests memory
+|requests:
+  cpu
+|Number of central processing units for requests, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1).
 |1Gi
-|16Gi
+|Proof of concept = 500m,
+Minimum deployment =1
 
-|Limits CPU
+|requests:
+  memory
+|Available memory for requests, based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi).
 |500m
-|1
-
-|Limits memory
-|1Gi
-|16Gi
+|Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+
+|limits:
+  cpu
+|Limit on number of central processing units, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1).
+|
+|Proof of concept = 500m,
+Minimum deployment =1
+
+|limits:
+  memory
+|Available memory limit based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi).
+|
+|Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+
+|
+4+|{asterisk} Each Elasticsearch node can operate with a lower memory setting though this is *not* recommended for production deployments. For production use, you should have no less than 16Gi allocated to each Pod by default, but preferably allocate as much as you can, up to 64Gi per Pod.
```

Comment:
Why up to 64Gi per pod?

Reply:
Maybe @jcantrill can provide more info what is the memory upper limit. Per https://www.elastic.co/guide/en/elasticsearch/reference/5.6/heap-size.html I thought it is 32gb

Reply:
The run.sh in the image halves the available memory which means 64G=32G of heap